core/math/basis.cpp

```diff
@@ -455,6 +455,8 @@ void Basis::get_rotation_axis_angle_local(Vector3 &p_axis, real_t &p_angle) cons
 }
 
 Vector3 Basis::get_euler(EulerOrder p_order) const {
+	const real_t epsilon = 0.00000025;
```

Comment:
A code comment should explain how this value came to be, so future devs won't have to look up this PR for context.